### PR TITLE
Small cleanup in hot path of NameResolution

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/fsharp/ConstraintSolver.fs [hunk 1]
```diff
@@ -648,15 +648,15 @@ let CheckWarnIfRigid (csenv:ConstraintSolverEnv) ty1 (r:Typar) ty =
...
+    
     DepthCheck ndeep m  ++ (fun () -> 
     match ty1 with 
     | TType_var r | TType_measure (MeasureVar r) ->
       // The types may still be equivalent due to abbreviations, which we are trying not to eliminate 
       if typeEquiv csenv.g ty1 ty then CompleteD else
 
       // The famous 'occursCheck' check to catch things like 'a = list<'a> 
-      if occursCheck csenv.g r ty then ErrorD (ConstraintSolverInfiniteTypes(denv,ty1,ty,m,m2)) else
+      if occursCheck csenv.g r ty then ErrorD (ConstraintSolverInfiniteTypes(csenv.DisplayEnv,ty1,ty,m,m2)) else
```
Comment (smoothdeveloper) [inline src/fsharp/ConstraintSolver.fs]: We may reindent and space the tuple

```fsharp
if occursCheck csenv.g r ty then
    ErrorD (ConstraintSolverInfiniteTypes(csenv.DisplayEnv, ty1, ty, m, m2))
else
// keep same indentation there, that fun () is huge
```

or at least keep the else on the next line

@dsyme what is up with style of spacing in tuples? I've seen some fortran codebase where those won't have any spaces between each argument in the list, I think it reads better with spacing but not sure what you prefer for the compiler codebase.

File: src/fsharp/NameResolution.fs [hunk 2]
```diff
@@ -696,13 +696,22 @@ let rec AddModuleOrNamespaceRefsToNameEnv g amap m root ad nenv (modrefs: Module
 and AddModuleOrNamespaceContentsToNameEnv (g:TcGlobals) amap (ad:AccessorDomain) m root nenv (modref:ModuleOrNamespaceRef) = 
      let pri = NextExtensionMethodPriority()
      let mty = modref.ModuleOrNamespaceType
```
Comment (smoothdeveloper) [inline src/fsharp/NameResolution.fs]: this bellow need careful review, can't easily assert if it does what it was doing before, but looks better!

File: src/fsharp/ConstraintSolver.fs (hunk 1 above)
Comment (dsyme) [inline src/fsharp/ConstraintSolver.fs]: Space-after-comma is preferred.  But as per my comment on another thread, that would go in a separate cleanup PR.

File: src/fsharp/NameResolution.fs (hunk 2 above)
Comment (dsyme) [inline src/fsharp/NameResolution.fs]: Checked it, looks ok to me

File: src/fsharp/NameResolution.fs [hunk 3]
```diff
@@ -696,13 +696,22 @@ let rec AddModuleOrNamespaceRefsToNameEnv g amap m root ad nenv (modrefs: Module
...
-     let tycons = mty.TypeAndExceptionDefinitions 
+     
+     let mutable state = { nenv with eDisplayEnv = nenv.eDisplayEnv.AddOpenModuleOrNamespace modref }
```
Comment (dsyme) [inline src/fsharp/NameResolution.fs]: I suggest like this is cleaner:

```fsharp
    let nenv = 
        let mutable state = ...
        for ...
        state
    ...

```

**Comment by dsyme** [comment]
This all looks good - added some comments above.

**Comment by KevinRansom** [comment]
@forki 
Thank you for taking care of this cleanup.

Kevin

**Pull request merged**